dvc/objects/file.py

```diff
@@ -1,7 +1,6 @@
 import logging
 
 from .errors import ObjectFormatError
-from .stage import get_file_hash
```

Comment:
This change seems unrelated.

Reply:
No, this is for preventing circular import (it now imports this after modules are finalized)

Reply:
Ah, makes sense. :+1: